Markdown/Default.sublime-keymap

```diff
@@ -0,0 +1,41 @@
+[
+    // Auto-pair asterisks
+    { "keys": ["*"], "command": "insert_snippet", "args": {"contents": "*$0*"}, "context":
+        [
+            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+            { "key": "selector", "operator": "equal", "operand": "text.html.markdown" },
+            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+            { "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|\\}|>|$)", "match_all": true },
+        ]
+    },
+    {
+        "keys": ["*"],
+        "command": "insert_snippet",
+        "args": {"contents": "*${0:$SELECTION}*"},
+        "context": [
+            { "key": "selector", "operator": "equal", "operand": "text.html.markdown" },
+            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+            { "key": "selection_empty", "operator": "equal", "operand": false, "match_all": true },
+        ]
+    },
+
+    // Auto-pair backticks
+    { "keys": ["`"], "command": "insert_snippet", "args": {"contents": "`$0`"}, "context":
```

Comment:
When starting or ending a code block you have to insert 3  backticks. This won't make it impossible but might be a bit unexpected that the first press of `` ` `` will insert two backticks already.

Reply:
Oh yes. I've set it so that the second asterisk or back tick is only added if there is at least one preceding non-whitespace character.